src/Projection/Projectionist/DefaultProjectionist.php

```diff
@@ -593,4 +590,92 @@ private function handleRetryProjections(ProjectionCollection $projections): void
             );
         }
     }
+
+    private function handleFromNowProjections(ProjectionCollection $projections): void
```

Comment:
```suggestion
    private function activateFromNowProjectionsWithLatestIndex(ProjectionCollection $projections): void
```
Maybe to be more clear?

Reply:
I will do this in a following PR as I have started building the Projectionist differently.